walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -415,7 +415,73 @@ public void noMandatoryAttendeesWithoutGaps() {
     request.addOptionalAttendee(PERSON_B);
 
     Collection<TimeRange> actual = query.query(events, request);
-    Collection<TimeRange> expected = Arrays.asList();
```

Comment:
I had to change the expected value of this test to be the entire day. Since this test contains only optional attendees, with their events filling up distinct portions of the day, the expected value should be the entire day. Since, that is the time slot that allows the maximum number of optional attendees (1) to attend the event in request.